vendor/ezurio/rs1xx-ext-multi-sensor.yaml

```diff
@@ -0,0 +1,183 @@
+
+# Hardware versions (optional, use when you have revisions)
+hardwareVersions:
+  - version: 'rev 4'
+    numeric: 1
+
+# Firmware versions (at least one is mandatory)
+firmwareVersions:
+  - # Firmware version
+    version: '6.1_20_6_16'
+    numeric: 1
+    # Corresponding hardware versions (optional)
+    hardwareVersions:
+      - 'rev 4'
+
+    # LoRaWAN Device Profiles per region
+    # Supported regions are EU863-870, US902-928, AU915-928, AS923, CN779-787, EU433, CN470-510, KR920-923, IN865-867, RU864-870
+    profiles:
+      EU863-870:
+        # Unique identifier of the profile (lowercase, alphanumeric with dashes, max 36 characters)
+        id: rs1xx-sensor-profile
+        lorawanCertified: false
+        codec: rs1xx-ext-multi-sensor-codec
+      US902-928:
+        id: rs1xx-sensor-profile
+        lorawanCertified: false
+        codec: rs1xx-ext-multi-sensor-codec
+      AU915-928:
+        id: rs1xx-sensor-profile
+        lorawanCertified: false
+        codec: rs1xx-ext-multi-sensor-codec
+      AS923:
+        id: rs1xx-sensor-profile
+        lorawanCertified: false
+        codec: rs1xx-ext-multi-sensor-codec
+
+# Sensors that this device features (optional)
+# Valid values are: accelerometer, altitude, auxiliary, barometer, battery, button, co2, distance, dust, gps, gyroscope,
+# humidity, light, link, magnetometer, moisture, ph, pir, proximity, rssi, snr, sound, temperature, tvoc, velocity,
+# vibration, water, wind direction and wind speed.
+sensors:
+  - temperature
+  - humidity
+  - proximity
+
+# Dimensions in mm (optional)
+# Use width, height, length and/or diameter
+dimensions:
+  width: 91
+  length: 116
+  height: 34
+
+# Weight in grams (optional)
+weight: 185
+
+# Battery information (optional)
+battery:
+  replaceable: true
+  type: AA
+
+# Operating conditions (optional)
+operatingConditions:
+  # Temperature (Celsius)
+  temperature:
+    min: -25
+    max: 50
+  # Relative humidity (fraction of 1)
+  relativeHumidity:
+    min: 0
+    max: 1
+
+# IP rating (optional)
+ipCode: IP65
+
+# Key provisioning (optional)
+# Valid values are: custom (user can configure keys), join server and manifest.
+keyProvisioning:
+  - custom
+  - join server
+
+# Key security (optional)
+# Valid values are: none, read protected and secure element.
+keySecurity: read protected
+
+# Product and data sheet URLs (optional)
+productURL: https://www.lairdconnect.com/wireless-modules/lorawan-solutions/sentrius-rs1xx-lora-enabled-sensors
+dataSheetURL: https://www.lairdconnect.com/documentation/sentrius-rs1xx-ext-openclosed-and-int-temphumidity-sensor-user-guide-v11
+resellerURLs:
+  - name: 'Digi-Key (RS1XX Open/Closed sensor)'
+    region:
+      - United States
+    url: https://www.digikey.com/en/products/detail/laird-connectivity-inc/455-00046/11587434
```

Comment:
Is this correct?

Reply:
@Jaime-Trinidad @johanstokking We are close to releasing our RS26x device and would like to have it added to the TTI Device Repository. This PR will need to be closed off before we can do this. Can I pick this one up and get anything outstanding closed off?

Reply:
@johanstokking Digikey links are unchanged.